pkg/sensors/tracing/generickprobe.go

```diff
@@ -56,33 +58,6 @@ func init() {
 	observer.RegisterEventHandlerAtInit(ops.MSG_OP_GENERIC_KPROBE, handleGenericKprobe)
 }
 
-const (
```

Comment:
why is this move needed? these seem to be bpf program related constants, is it breaking build?

Reply:
Yes, they break build in an otherwise platform independent code listing